src/task.c

```diff
@@ -640,72 +661,69 @@ task_enqueue(task_system *ts,
       }
    }
    platform_condvar_signal(&group->cv);
-   return platform_condvar_unlock(&group->cv);
+   return task_group_unlock(group);
 }
 
+/*
+ * Run a task if the number of waiting tasks is at least min_backlog.
```

Comment:
The comment is no longer consistent with the introduction of the new `queue_scale_percent` argument.

I found it a bit difficult to ferret out the semantic of this variable. 

Can you please add some comments here explaining what this %age variable means? And give some guidance on what it should be set to, if ever someone wishes to change it from its current default of 100% (`TEST_CONFIG_DEFAULT_QUEUE_SCALE_PERCENT`)?